File: URI_1012_area.js
const lines = ["3.0 4.0 5.2"];

let inputArr = lines[0].split(" ");
let a = Number(inputArr[0]);
let b = Number(inputArr[1]);
let c = Number(inputArr[2]);
const PI = 3.14159;

let triangleArea = a * c / 2;
let circleArea = PI * c * c;
let trapeziumArea = (a + b) * c / 2;
let squareArea = b * b;
let rectangleArea = a * b;

console.log("TRIANGULO: " + triangleArea.toFixed(3));
console.log("CIRCULO: " + circleArea.toFixed(3));
console.log("TRAPEZIO: " + trapeziumArea.toFixed(3));
console.log("QUADRADO: " + squareArea.toFixed(3));
console.log("RETANGULO: " + rectangleArea.toFixed(3));

File: URI_1012_area.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class URI_1012_area {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double a = scanner.nextDouble();
		double b = scanner.nextDouble();
		double c = scanner.nextDouble();
		final double PI = 3.14159;

		double triangleArea = a * c / 2;
		double circleArea = PI * c * c;
		double trapeziumArea = (a + b) * c / 2;
		double squareArea = b * b;
		double rectangleArea = a * b;

		DecimalFormat df = new DecimalFormat("0.000");
		System.out.println("TRIANGULO: " + df.format(triangleArea));
		System.out.println("CIRCULO: " + df.format(circleArea));
		System.out.println("TRAPEZIO: " + df.format(trapeziumArea));
		System.out.println("QUADRADO: " + df.format(squareArea));
		System.out.println("RETANGULO: " + df.format(rectangleArea));
		
		scanner.close();
	}
}
